packages/std/src/storage_keys/length_prefixed.rs

```diff
@@ -38,6 +40,14 @@ fn encode_length(namespace: &[u8]) -> [u8; 2] {
     [length_bytes[2], length_bytes[3]]
 }
 
+#[inline]
+#[allow(unused)]
+fn concat(namespace: &[u8], key: &[u8]) -> Vec<u8> {
```

Comment:
It was never public so it can be removed eventually.

I think instead of making it public we better make a version of `namespaces_with_key` available.

Reply:
I removed concat as I consider it too trivial to be maintained in a library. `namespace_with_key` combines namespace and key encoding now.